hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestAsyncNonMetaRegionLocator.java

```diff
@@ -390,6 +404,98 @@ public RegionLocations getRegionLocations(TableName tableName, int replicaId, bo
     });
   }
 
+  @Test
+  public void testRegionReplicaRitRecovery() throws Exception {
+    int startRowNum = 11;
+    int rowCount = 60;
+
+    final TableName tableName = TableName.valueOf(name.getMethodName());
+    final ProcedureExecutor<MasterProcedureEnv> procExec = getMasterProcedureExecutor();
+    TEST_UTIL.getAdmin().createTable(
+      TableDescriptorBuilder.newBuilder(tableName)
+          .setColumnFamily(ColumnFamilyDescriptorBuilder.of(FAMILY)).setRegionReplication(2)
+          .build());
+    TEST_UTIL.waitUntilAllRegionsAssigned(tableName);
+    RegionLocations locs = new Locator() {
```

Comment:
Seems we are not using this _locs_ variable anywhere else in the test. Is this really needed? Otherwise we can remove lines #419 to #433

Reply:
ok will remove this unused variable stuff.

Reply:
Had removed it on the last commit.